### PR TITLE
feat: articles about new LFT with streamyard

### PR DESCRIPTION
**Pull request by Slashgear**
New article about the way we are now streaming and recording our LFT with streamyard.

it follows @pmartin last article about LFT

=> https://pr-305.dsvmt7xpjktgx.amplifyapp.com/2022/04/09/how-we-simplify-our-lft-broadcast.html

### CODE REVIEW

File: _posts/2022-04-09-how-we-simplify-our-lft-broadcast.md [hunk 1]
```diff
@@ -0,0 +1,128 @@
...
+
+![Youtube video replay screenshot](/images/posts/lft-streamyard/replay.png)
+
+## Next steps
+
+We don't want to stop there.
+Every two months, we organize a LFT day at Bedrock: our Last Friday Talks.
+
+For the next editions, we will try to do even better.
+We are working on the subject of replaying our LFT in public on the Youtube channel in order to offer it to everyone.
```
Comment (pmartin) [inline _posts/2022-04-09-how-we-simplify-our-lft-broadcast.md:124]:
```suggestion
We are working on the matter of sharing some talks in public on our Youtube channel, so more people in our communities can learn from them.
```

File: _posts/2022-04-09-how-we-simplify-our-lft-broadcast.md [hunk 2]
```diff
@@ -0,0 +1,128 @@
...
+In order to simplify the setup, we will try to put in place a more fixed table to avoid wiring and moving furniture.
+We also wish to propose and train our employees to the use of this setup in order to allow us to host meetups and conferences in the best conditions.
+
+Your company or association also proposes this kind of conferences, how do you manage broadcasting and recording?
```
Comment (pmartin) [inline _posts/2022-04-09-how-we-simplify-our-lft-broadcast.md:128]:
```suggestion
Now, it's your turn: if your company or user-group does this kind of talks, how do you manage broadcasting and recording?
```

File: _posts/2022-04-09-how-we-simplify-our-lft-broadcast.md [hunk 3]
```diff
@@ -0,0 +1,128 @@
...
+
+As a reminder, here is the organization of our auditorium during the live broadcast of our LFT.
+The room is large enough to accommodate all Bedrockers who wish to attend in person the presentations of their colleagues. 
+The remote speakers have their conference broadcasted on the two screens of the room.
+
+![Auditorium plan](/images/posts/lft-streamyard/plan.jpg)
+
+![Control desk](/images/posts/lft-streamyard/control-desk.jpg)
+
+![Auditorium room](/images/posts/lft-streamyard/amphi-bedrock.png)
```
Comment (Slashgear) [inline _posts/2022-04-09-how-we-simplify-our-lft-broadcast.md:76]:
```suggestion
![Auditorium room](/images/posts/lft-streamyard/amphi-bedrock.jpg)
```

File: _posts/2022-04-09-how-we-simplify-our-lft-broadcast.md [hunk 4]
```diff
@@ -0,0 +1,128 @@
...
+
+### New setup
+
+As a reminder, here is the organization of our auditorium during the live broadcast of our LFT.
+The room is large enough to accommodate all Bedrockers who wish to attend in person the presentations of their colleagues. 
+The remote speakers have their conference broadcasted on the two screens of the room.
+
+![Auditorium plan](/images/posts/lft-streamyard/plan.jpg)
+
+![Control desk](/images/posts/lft-streamyard/control-desk.jpg)
```
Comment (Slashgear) [inline _posts/2022-04-09-how-we-simplify-our-lft-broadcast.md:74]:
```suggestion
![Control desk](/images/posts/lft-streamyard/control-desk.png)
```

File: _posts/2022-04-09-how-we-simplify-our-lft-broadcast.md [hunk 5]
```diff
@@ -0,0 +1,127 @@
...
+As a reminder, our internal conferences or meetups are called **LFT**. 
+If you want to know more about our _Last Friday Talk_ and the motivations that push us to do it, we invite you to read the above-mentioned article.
+
+After the success of this broadcasting, the (voluntary) organization team started thinking about how to make it simpler. 
+_It was already a challenge in itself!_
+If you read [this part of the article](/2021/10/14/live-streaming-lft.html#Some%20things%20to%20improve), you can see that we already had some ideas for improvements.
+
+## Improvement areas
+
+In the previous version, you could see that a very important quantity of material was necessary (meters of various cables, multi plugs, a camera, etc...)
```
Comment (Meroje) [inline _posts/2022-04-09-how-we-simplify-our-lft-broadcast.md:25]:
```suggestion
In the previous version, you could see that a very important quantity of material was necessary (meters of various cables, multiple outlets, a camera, etc...)
```

File: _posts/2022-04-09-how-we-simplify-our-lft-broadcast.md [hunk 6]
```diff
@@ -0,0 +1,127 @@
...
+![Stream yard screenshot with comment from chat](/images/posts/lft-streamyard/comment-from-chat.png)
+
+The positive points of Streamyard:
+
+- Allows us to have several people in the control room at the same time
+- Streams in 1080p
+- Personalization: manage banners, chat questions, music on hold,...
+- Handles multi-speakers management
+- Multi stream Youtube / Facebook / Twitter / Linkedin / Video recording
+- A free mode allows one to test it before taking out their credit card.
```
Comment (Meroje) [inline _posts/2022-04-09-how-we-simplify-our-lft-broadcast.md:64]:
```suggestion
- Re-stream to Youtube / Facebook / Twitter / Linkedin / Video recording
- A free trial mode allows one to test it before taking out their credit card.
```

File: _posts/2022-04-09-how-we-simplify-our-lft-broadcast.md [hunk 7]
```diff
@@ -0,0 +1,127 @@
...
+What has changed since we switched to Streamyard is mainly related to the way we capture the image and sound of what is displayed on the screen.
+Previously, we were using an Elgato HD60S+ device, to capture the HDMI output from the speaker broadcast. 
+However, we had to try several times to get it to work each time we switched speakers. Not fun at all.
+
+Now, with Streamyard, every time we switch speaker, we just need to:
+- share the link of the "Streamyard broadcast"
+- The speaker joins the stream (and switches off her microphone and webcam)
+- Connect the HDMI cable of the speaker's computer
+- The speaker displays her screen via the room's projectors
+- The control room puts the RF microphone on the speaker(s)
```
Comment (Meroje) [inline _posts/2022-04-09-how-we-simplify-our-lft-broadcast.md:87]:
```suggestion
- The control room operator puts the RF microphone on the speaker(s)
```

File: _posts/2022-04-09-how-we-simplify-our-lft-broadcast.md [hunk 8]
```diff
@@ -0,0 +1,127 @@
...
+Also, by going through the youtube chat, we can share questions directly on the screen and on the replay.
+
+The LFT replays are also available on Bedrock's Youtube channel in a private way accessible to all employees.
+
+![Youtube video replay screenshot](/images/posts/lft-streamyard/replay.png)
+
+## Next steps
+
+We don't want to stop there.
+Every two months, we organize a LFT day at Bedrock: our Last Friday Talks.
```
Comment (Meroje) [inline _posts/2022-04-09-how-we-simplify-our-lft-broadcast.md:120]: This feels like a repeat

File: _posts/2022-04-09-how-we-simplify-our-lft-broadcast.md [hunk 9]
```diff
@@ -0,0 +1,127 @@
...
+
+For the next editions, we will try to do even better.
+We are working on the matter of sharing some talks in public on our Youtube channel, so more people in our communities can learn from them.
+In order to simplify the setup, we will try to put in place a more fixed table to avoid wiring and moving furniture.
+We also wish to propose and train our employees to the use of this setup in order to allow us to host meetups and conferences in the best conditions.
```
Comment (Meroje) [inline _posts/2022-04-09-how-we-simplify-our-lft-broadcast.md:125]: I think we can tick this box :)

File: _posts/2022-04-09-how-we-simplify-our-lft-broadcast.md [hunk 10]
```diff
@@ -0,0 +1,127 @@
...
+
+Now, it's your turn: if your company or user-group does this kind of talks, how do you manage broadcasting and recording?
```
Comment (Meroje) [inline _posts/2022-04-09-how-we-simplify-our-lft-broadcast.md:127]: Where are readers supposed to answer this CTA ?

**Pull request merged**